simapp/v2/app_di.go

```diff
@@ -92,7 +91,6 @@ func NewSimApp[T transaction.Tx](
 	logger log.Logger,
 	viper *viper.Viper,
 ) *SimApp[T] {
-	viper.Set(serverv2.FlagHome, DefaultNodeHome) // TODO possibly set earlier when viper is created
```

Comment:
Can we re-add this in newApp and appExport in `commands.go`?

Reply:
not sure if I understand the reason to set it, by default is already set the DefaultNode. Do we want to enforce nobody can set their own FlagHome?

Reply:
We do some flag parsing in init() in app.go (required as we need to parse them as early as possible)